op-chain-ops/genesis/check.go

```diff
@@ -59,9 +59,6 @@ var (
 			common.Hash{31: 0x97}: common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000001"),
 			// Slot 0xcc (204) is xDomainMsgSender
 			common.Hash{31: 0xcc}: common.HexToHash("0x000000000000000000000000000000000000000000000000000000000000dead"),
-			// EIP-1967 storage slots
-			AdminSlot:          common.HexToHash("0x0000000000000000000000004200000000000000000000000000000000000018"),
```

Comment:
The messenger is still behind a proxy - its 51 and 151 that no longer exist

See:
- https://github.com/ethereum-optimism/optimism/blob/4071f64e5dd582beaa28c393b20757193361a4ca/packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol#L77
- https://github.com/ethereum-optimism/optimism/blob/4071f64e5dd582beaa28c393b20757193361a4ca/packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol#L45